Improvement to compiler plugin page (#6160)
detekt/detekt: website/docs/gettingstarted/compilerplugin.mdx

sidebar_position: 7
---

> You can integrate detekt in your project using the Detekt Compiler Plugin instead of the classic [Detekt Gradle Plugin](/docs/gettingstarted/gradle). Detekt offers a compiler plugin for K1 which allows you to run detekt as part of the Kotlin compilation process using [type resolution](/docs/gettingstarted/type-resolution). This allows you to run detekt on your code without having separate tasks to invoke and results in much faster execution of detekt, especially if you have been using type resolution with the Gradle plugin.

Reviewer: ```suggestion
You can integrate detekt in your project using the Detekt Compiler Plugin instead of the classic [Detekt Gradle Plugin](/docs/gettingstarted/gradle). Detekt offers a compiler plugin for K1 which allows you to run detekt as part of the Kotlin compilation process using [type resolution](/docs/gettingstarted/type-resolution). This allows you to run detekt on your code without having separate tasks to invoke and results in much faster execution of detekt, especially if you use type resolution with the Gradle plugin.
```
